testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/DeployAllDomainTestCase.java

```diff
@@ -447,4 +448,225 @@ public void testLegacyEnableSingleAppDeployment() throws CommandLineException, I
         checkExist(result, cliTestApp2War.getName(), ADDED, ctx);
         checkExist(result, cliTestAppEar.getName(), ENABLED, ctx);
     }
+
+    /**
+     * Disabling all deployed application deployments between all server groups.
+     * Separate test for check status of issue WFCORE-3562 by Aesh command.
+     * Concentration is for disabling all deployments with all group by Aesh commands.
+     *
+     * @see <a href="https://issues.jboss.org/browse/WFCORE-3562">WFCORE-3562</a>
+     */
+    @Test
+    public void testDisableAllAppDeploymentAllGroup() throws CommandLineException, IOException {
+        // Step 1) Deploy applications deployments to defined server groups
+        ctx.handle("deployment deploy-file --server-groups=" + sgOne + ' ' + cliTestApp1War.getAbsolutePath());
+        ctx.handle("deployment deploy-file --server-groups=" + sgOne + ' ' + cliTestAnotherWar.getAbsolutePath());
+        ctx.handle("deployment deploy-file --server-groups=" + sgTwo + ' ' + cliTestApp2War.getAbsolutePath());
+        ctx.handle("deployment deploy-file --server-groups=" + sgTwo + ',' + sgOne + ' ' + cliTestAppEar.getAbsolutePath());
+
+
+        // Step 2) Disabling two selected applications deployments
```

Comment:
"applications deployments" ==> "deployments". It applies to some other locations in this test.

Reply:
Is just the word 'deployments'? Ok, replaced.